src/scripts/app.js

```diff
@@ -1 +1,22 @@
 console.log('hello');
```

Comment:
This code is only for menu in header section. The better way will be create individual file mobile header.js or sth like this.

Reply:
done as suggested